portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java

```diff
@@ -31,8 +31,9 @@
 
 @WebServlet("/login-status")
 public class LoginServlet extends HttpServlet {
+    Gson gson = new Gson();
```

Comment:
Whenever you put a field in a class, you should default to `private final` unless you have a good reason to do otherwise.

Reply:
done